lib/install.go

```diff
@@ -49,6 +49,29 @@ func initialize() {
 
 }
 
+// get install path variable value  (windows os runtime support)
+func getInstallPath() string {
+	return string(os.PathSeparator) + installPath + string(os.PathSeparator)
+}
+
+// get versioned install filename (windows os runtime support)
+func getVersionedInstallFileName(tfversion string) string {
+	if runtime.GOOS == "windows" {
+		return getInstallLocation() + installVersion + tfversion + ".exe"
```

Comment:
```suggestion
		return filepath.Join(getInstallLocation(), installVersion+ tfversion + ".exe")
```

Reply:
PertsevRoman  are you able to amend this?